sn_client/src/wallet.rs

```diff
@@ -35,6 +35,7 @@ pub struct WalletClient {
 
 impl WalletClient {
     /// Create a new wallet client.
+    #[doc = include_str!("docs/wallet/new.md")]
```

Comment:
I don't think there's any reason to move the docs to another file here. 

As I mentioned, there's an extra burden to move to update these docs, so to my mind unless they are _huge_ we'd want to keep them in the same file.